src/api/app/controllers/person/token_controller.rb

```diff
@@ -14,7 +14,12 @@ def index
     def create
       authorize @user, :update?
 
-      pkg = (Package.get_by_project_and_name(params[:project], params[:package]) if params[:project] || params[:package])
+      unless params[:project] && params[:package]
+        render_error status: 400, errorcode: 'missing_parameters', message: 'The package and project parameters must be provided.'
```

Comment:
```suggestion
       raise MissingParameterError, 'The package and project parameters must be provided.'
```

Reply:
Let me know if something is not clear

Reply:
Everything clear, thanks for the suggestion.